### PR TITLE
Li/add notification examples

### PR DESCRIPTION
**Pull request by li-vendia**
Added all possible notification guides. Merge ready when review is completed.

This branch is supplement for notification info sheet. It will include all notification options and along with different ways of setting them up.

### CODE REVIEW

File: features/README.md [hunk 1]
```diff
@@ -7,10 +7,12 @@
...
+| [GraphQL Enhancements](share/graphql-enhancements/README.md)  | Golang, Python, JavaScript, GraphQL | Explore GraphQL enhancements (like conditional puts and transactions) to conventional GraphQL functionality using several programming languages | 
+| [GraphQL Filters](share/graphql-filters/README.md)            | GraphQL                             | Explore the GraphQL filter syntax in depth                                                                                                      | 
+| [Smart Contracts](share/smart-contracts/README.md)            | JavaScript, GraphQL                 | Explore a set of smart contracts that are used for data validation, computation, and enrichment                                                 |
+| [Notifications](share/notifications/README.md)                | GraphQL                             | Explore ways to receieve notifications from Vendia through both UI and graphQL                                                                  |
```
Comment (james-vendia) [inline features/README.md:17]: Also...is it _really_ receiving notifications through GraphQL?

File: features/share/notifications/README.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
+<p align="center">
+  <a href="https://vendia.net/">
+    <img src="https://www.vendia.net/images/logo/black.svg" alt="vendia logo" width="250px">
+  </a>
+</p>
+
+# Purpose
+This README.md is meant to help you setup your environment to test out all of the notification options Vendia Share offers. **Please complete all the prerequisites before going to each example links**.
```
Comment (james-vendia) [inline features/share/notifications/README.md:8]:
```suggestion
This README.md is meant to help you setup your environment to test out all of the notification options Vendia Share offers.

**Please complete all the prerequisites before going to each example links**.
```

File: features/share/notifications/README.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+
+# Prerequisites
+* Know what Vendia Notifications do. Or check our docs [here](https://www.vendia.com/docs/share/integrations#outbound)
```
Comment (james-vendia) [inline features/share/notifications/README.md:11]:
```suggestion
* Know what Vendia Notifications do. Check out the [docs] (https://www.vendia.com/docs/share/integrations#outbound)
```

File: features/share/notifications/README.md [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+* Create a [Vendia Share account](https://share.vendia.net/)
+* Create a [Amazon Web Services account](https://aws.amazon.com/free)
+* Install [Node.js](https://nodejs.org/en/download/)
+* Install the Vendia Share [Command Line Interface (CLI)](https://www.vendia.net/docs/share/cli)
+* Install [Git Client](https://git-scm.com/downloads)
+
+# Uni Creation
+
+Unis can be created through both [Vendia Share UI](https://share.vendia.net/) or our [Share CLI](https://www.vendia.com/docs/share/cli).
```
Comment (james-vendia) [inline features/share/notifications/README.md:20]:
```suggestion
Unis can be created through both the [Vendia Share UI](https://share.vendia.net/) or the [Vendia Share CLI](https://www.vendia.com/docs/share/cli).
```

File: features/share/notifications/README.md [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+
+## Creation Through UI
+1. login to [Vendia share](https://share.vendia.net).
```
Comment (james-vendia) [inline features/share/notifications/README.md:23]:
```suggestion
1. Log in to [Vendia share](https://share.vendia.net).
```

Comment (james-vendia) [reply features/share/notifications/README.md:23]: Double check capitalization and phrasing consistency throughout the READMEs.

File: features/share/notifications/README.md [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+
+Wait about 5 minutes for the Uni to reach a `Running` state.
+
+***Note:*** If you take a look at the information we have inside the `registration.json` file and the `schema.json` file, you'll notice that we are essentially doing the exactly same thing as the UI creation step. Notice that all configurations provided are identical.
+
+# Pick the Tool of Your Preference
+Once all the notes are in `running` status, choose one of the examples below and continue.
+
+
+## Success Notifications
```
Comment (james-vendia) [inline features/share/notifications/README.md:70]: I get your intent with the `Programming Language` column but I'd remove it.  Most of the examples here are not really programs (in the source code sense) but rather Share config being applied to a Node.

For `Description`, check for consistency in verbs and nouns being used.  These are really all the same thing, but just through different transports - the descriptions should be the same as well.

**Review by james-vendia — CHANGES_REQUESTED** [review]
Comments inline for your review.  Two others notes (which I can't inline):

1.) Please rename `image` to `img` to match what's already in the repo (see - [features/share/notifications/image/error/click-error-notification.png](https://github.com/vendia/examples/tree/main/integrations/security/saml-example/img))

2.) Somewhere we should highlight (and maybe I missed it in a readme) that Success Notifications go to ALL nodes in a Uni, while Error Notifications go to ONLY the node to which the mutation was sent.  This has been a point of confusion for some of our customers.  Would love a way to guide them, with that context called out.

File: features/README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (li-vendia) [reply features/README.md:17]: I do see that our notification was written under an integration section. But these guide didn't dive deep into how the pieces outside of Vendia work and essentially focused only on our "Feature" notification. So I believe it belongs here.

Comment (li-vendia) [reply features/README.md:17]: Nope. That's just my bad English. Let me correct that.

File: features/share/notifications/README.md (hunk 6 above)
Comment (li-vendia) [reply features/share/notifications/README.md:72]: removed and changed all the descriptions consistently to Receive notification with <tool-name>. Though it sounds boring but it does look sharp. :)

File: features/share/notifications/README.md [hunk 7]
```diff
@@ -0,0 +1,92 @@
...
+Notifications emitted when the node commits a new block. Includes a summary of the transactions included in the block.
+
+| Example                                                          | Description                                 |
+|:-----------------------------------------------------------------|:--------------------------------------------|
+| [Email](success-notification/email/README.md)                    | Receive notifications with e-mail           |
+| [Kinesis Firehoses](success-notification/aws-firehose/README.md) | Receive notifications with Kinese Firehoses |
```
Comment (james-vendia) [inline features/share/notifications/README.md:78]:
```suggestion
| [Kinesis Data Firehose](success-notification/aws-firehose/README.md) | Receive notifications with Kinesis Data Firehose |
```

File: features/share/notifications/success-notification/aws-firehose/README.md [hunk 8]
```diff
@@ -0,0 +1,154 @@
+<p align="center">
+  <a href="https://vendia.net/">
+    <img src="https://www.vendia.net/images/logo/black.svg" alt="vendia logo" width="250px">
+  </a>
+</p>
+
+# Receive Success Notifications Through Kinese Firehoses
```
Comment (james-vendia) [inline features/share/notifications/success-notification/aws-firehose/README.md:7]:
```suggestion
# Receive Success Notifications Through Kinesis Data Firehose
```

File: features/share/notifications/success-notification/aws-firehose/README.md [hunk 9]
```diff
@@ -0,0 +1,154 @@
...
+
+## Purpose
+This is a guide on how to set up success notifications on aws firehose. Note that all notification examples include 2 parts: UI Setup & GraphQL setup. Readers don't have to go through both cases.
```
Comment (james-vendia) [inline features/share/notifications/success-notification/aws-firehose/README.md:10]:
```suggestion
This is a guide on how to set up success notifications on Amazon Kinesis Data Firehose. Note that all notification examples include 2 parts: UI Setup & GraphQL setup. Readers don't have to go through both cases.
```

File: features/share/notifications/success-notification/aws-firehose/README.md [hunk 10]
```diff
@@ -0,0 +1,154 @@
...
+
+# Prerequisites
+* Completed the setup in accordance with this [README.md](../../README.md)
+* AWS account with enough access to create firehose and adding subscription for SNS
+* AWS CLI v2.8.7 Note: Versions are not hard requirement but could potentially cause issues.
+* Basic knowledge of AWS firehose, S3, and IAM.
```
Comment (james-vendia) [inline features/share/notifications/success-notification/aws-firehose/README.md:16]:
```suggestion
* Basic knowledge of Amazon Kinesis Data Firehose, S3, and IAM.
```

**Review by james-vendia — CHANGES_REQUESTED** [review]
Small changes, mainly on AWS/Amazon service naming.

I like where this is going.  Nearly complete.

File: features/share/notifications/success-notification/aws-firehose/README.md (hunk 9 above)
Comment (li-vendia) [reply features/share/notifications/success-notification/aws-firehose/README.md:10]: I would love to hear story about that.

**Review by james-vendia — APPROVED** [review]
Ready to merge @li-vendia

**Pull request merged**